File: Programmers/12901.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(int a, int b) {
    string answer = "";
    int result = 0;

    vector<string> vec = {"THU","FRI","SAT","SUN","MON","TUE","WED"};
    vector<int> month = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};

    for(int i = 1; i < a; i++) {
        result += month[i];
    }
    result += b;

    answer = vec[result%7];

    return answer;
}

int main() {

    cout << solution(5, 24) << endl;
    return 0;
}

File: Python/programmers0-1/12901.py
def solution(a, b):
    weekdays = {1: "FRI", 2: "SAT", 3:"SUN", 4:"MON", 5:"TUE", 6:"WED", 0:"THU"}
    months = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    return weekdays[(sum(months[0:a-1]) + b) % 7]